app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,24 @@
  *= require_tree .
  *= require_self
  */
+body {
+}
+ .home {
+  max-height: 100vh;
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #e6e6e6;
+  padding-bottom: 50px;
```

Comment:
sort styles alphaneticallly

Reply:
Select properties and do Ctrl + Shift + P to open the command palette and then search for "sort"